src/proposal/mod.rs

```diff
@@ -245,13 +245,14 @@ pub fn get_subscribers(proposal_body: &ProposalBodyV2) -> Vec<String> {
 
 pub fn default_categories() -> Vec<String> {
     vec![
-        String::from("DevDAO Operations"),
-        String::from("Decentralized DevRel"),
-        String::from("NEAR Campus"),
-        String::from("Marketing"),
-        String::from("Events"),
-        String::from("Tooling & Infrastructures"),
-        String::from("Other"),
+        String::from("Bounty"),
+        String::from("Bounty booster"),
+        String::from("Event sponsorship"),
+        String::from("Events team"),
+        String::from("Hackathon"),
+        String::from("Hackbox"),
+        String::from("Meetup"),
+        String::from("Travel expenses"),
```

Comment:
I think there is no sane defaults anymore, so I would drop the whole function completely.